apps/heureka/src/components/common/ServiceImages/ImagesDataRows/ImagesDataRow.tsx

```diff
@@ -4,52 +4,63 @@
  */
 
 import React from "react"
-import { ServiceImageVersion } from "../../../Services/utils"
+import { ServiceImage } from "../../../Services/utils"
 import { DataGridRow, DataGridCell, Button, Icon, Stack } from "@cloudoperators/juno-ui-components"
 import { SeverityCount } from "../../SeverityCount"
 
-type ImageVersionsDataRowProps = {
-  version: ServiceImageVersion
+type ImagesDataRowProps = {
+  version: ServiceImage
   selected: boolean
   displayDetailsButton?: boolean
   onItemClick: React.MouseEventHandler<HTMLDivElement>
   onDetailClick: React.MouseEventHandler<HTMLButtonElement | HTMLAnchorElement>
 }
 
-export const ImageVersionsDataRow = ({
+export const ImagesDataRow = ({
   version,
   selected = false,
   displayDetailsButton = true,
   onItemClick,
   onDetailClick,
-}: ImageVersionsDataRowProps) => {
+}: ImagesDataRowProps) => {
   return (
-    <DataGridRow onClick={onItemClick} className={`cursor-pointer ${selected ? "active" : ""}`}>
+    <DataGridRow onClick={onItemClick} className="cursor-pointer" isSelected={selected}>
       <DataGridCell className="service-image-versions-cell">
         <Stack gap="1" direction="vertical">
           <Stack gap="0.5" direction="vertical">
             <span>{version.repository}</span>
-            <span className="text-sm text-theme-light">{version.version}</span>
+            {version.versionsCount !== undefined && version.versionsCount > 0 && (
+              <span className="text-sm text-theme-light">
+                {version.versionsCount} version{version.versionsCount !== 1 ? "s" : ""}
+              </span>
+            )}
           </Stack>
           <Stack gap="1" alignment="center">
-            <a
-              href={`https://${version.ccrn}`}
-              target="_blank"
-              rel="noopener noreferrer"
-              className="hover:underline text-sm"
-              onClick={(e) => e.stopPropagation()}
-            >
-              <Stack gap="1.5" alignment="center">
-                <Icon icon="openInNew" size="16" color="global-text" />
-                <span>Image registry</span>
-              </Stack>
-            </a>
+            {version.imageRegistryUrl && (
+              <a
```

Comment:
We should keep in mind that when using raw components, we also need to address accessibility until a proper UI component is available:
```
aria-label={Open image registry for ${version.repository} (opens in a new tab)} 
title={Open image registry for ${version.repository} (opens in a new tab)}
```